src/treelearner/serial_tree_learner.h

```diff
@@ -180,6 +189,13 @@ class SerialTreeLearner: public TreeLearner {
   std::vector<int> ordered_bin_indices_;
   bool is_constant_hessian_;
   std::unique_ptr<CostEfficientGradientBoosting> cegb_;
+
+  std::vector<std::vector<double> > dummy_min_constraints;
+  std::vector<std::vector<double> > min_constraints;
+  std::vector<std::vector<double> > dummy_max_constraints;
+  std::vector<std::vector<double> > max_constraints;
```

Comment:
where use these ?

Reply:
My bad, these were used before refactoring, and I forgot to remove them.